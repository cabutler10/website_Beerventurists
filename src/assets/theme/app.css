html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
