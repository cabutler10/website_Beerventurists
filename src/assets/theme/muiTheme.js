import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
