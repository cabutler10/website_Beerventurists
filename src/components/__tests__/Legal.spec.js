import React from 'react'
import { render } from 'react-testing-library'
import Legal from '../Legal'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('Legal', () => {
  it('renders correctly', () => {
    const mockProps = {
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <Legal {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
