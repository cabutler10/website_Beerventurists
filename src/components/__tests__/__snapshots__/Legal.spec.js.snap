// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legal renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-container-uo3ybq"
        id="legal"
      >
        <h5
          class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-1j3kb4i"
        >
          imprint.imprint
        </h5>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Eisbach Riders 
          <br />
          Robin Eisenhardt & Michael Schmidt GbR 
          <br />
          Lothstr. 72 
          <br />
          80797 München 
          <br />
          Deutschland 
          <br />
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Tel.: 0176/69019542 
          <br />
          E-Mail: business@eisbach-riders.com
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Vertretungsberechtigte Gesellschafter: Robin Eisenhardt, Michael Schmidt
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Umsatzsteuer-Identifikationsnummer gemäß §27 a Umsatzsteuergesetz: DE 320071401 Plattform der EU-Kommission zur 
          <br />
          Online-Streitbeilegung:
           
          <a
            href="https://ec.europa.eu/consumers/odr"
          >
            https://ec.europa.eu/consumers/odr
          </a>
          Wir sind zur Teilnahme an einem Streitbeilegungsverfahren vor einer Verbraucherschlichtungsstelle weder verpflichtet noch bereit.
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Verantwortliche/r i.S.d. § 55 Abs. 2 RStV: Robin Eisenhardt, Michael Schmidt, Lothstr. 72, 80797, München
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-1j3kb4i"
        >
          imprint.terms
        </h6>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title1
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms1_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms1_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms1_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title2
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_3
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_4
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_5
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_6
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms2_7
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title3
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms3_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms3_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title4
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms4_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms4_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms4_3
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms4_4
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title5
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms5_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms5_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms5_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title6
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms6_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title7
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms7_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms7_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms7_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title8
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms8_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title9
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms9_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms9_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-1j3kb4i"
        >
          imprint.title10
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms10_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title11
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_3
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_4
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_5
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms11_6
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          imprint.title12
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms12_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          Eisbach Riders 
          <br />
          Robin Eisenhardt & Michael Schmidt GbR 
          <br />
          Lothstr. 72 
          <br />
          80797 München 
          <br />
          Deutschland 
          <br />
          E-Mail: business@eisbach-riders.com
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
        >
          imprint.terms12_2
        </p>
        <h5
          class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-1j3kb4i"
        >
          imprint.credits
        </h5>
        <a
          href="https://unsplash.com/@thefoxicon?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Mike Fox"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Mike Fox
          </span>
        </a>
        <a
          href="https://unsplash.com/@jeremybishop?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Jeremy Bishop"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Jeremy Bishop
          </span>
        </a>
        <a
          href="https://unsplash.com/@timmarshall?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Tim Marshall"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Tim Marshall
          </span>
        </a>
        <div>
          Icons made by
           
          <a
            href="https://www.freepik.com/"
            title="Freepik"
          >
            Freepik
          </a>
           
          from
           
          <a
            href="https://www.flaticon.com/"
            title="Flaticon"
          >
            www.flaticon.com
          </a>
           
          is licensed by
           
          <a
            href="http://creativecommons.org/licenses/by/3.0/"
            target="_blank"
            title="Creative Commons BY 3.0"
          >
            CC 3.0 BY
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-container-uo3ybq"
      id="legal"
    >
      <h5
        class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-1j3kb4i"
      >
        imprint.imprint
      </h5>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Eisbach Riders 
        <br />
        Robin Eisenhardt & Michael Schmidt GbR 
        <br />
        Lothstr. 72 
        <br />
        80797 München 
        <br />
        Deutschland 
        <br />
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Tel.: 0176/69019542 
        <br />
        E-Mail: business@eisbach-riders.com
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Vertretungsberechtigte Gesellschafter: Robin Eisenhardt, Michael Schmidt
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Umsatzsteuer-Identifikationsnummer gemäß §27 a Umsatzsteuergesetz: DE 320071401 Plattform der EU-Kommission zur 
        <br />
        Online-Streitbeilegung:
         
        <a
          href="https://ec.europa.eu/consumers/odr"
        >
          https://ec.europa.eu/consumers/odr
        </a>
        Wir sind zur Teilnahme an einem Streitbeilegungsverfahren vor einer Verbraucherschlichtungsstelle weder verpflichtet noch bereit.
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Verantwortliche/r i.S.d. § 55 Abs. 2 RStV: Robin Eisenhardt, Michael Schmidt, Lothstr. 72, 80797, München
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-1j3kb4i"
      >
        imprint.terms
      </h6>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title1
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms1_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms1_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms1_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title2
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_3
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_4
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_5
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_6
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms2_7
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title3
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms3_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms3_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title4
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms4_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms4_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms4_3
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms4_4
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title5
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms5_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms5_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms5_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title6
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms6_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title7
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms7_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms7_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms7_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title8
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms8_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title9
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms9_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms9_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-1j3kb4i"
      >
        imprint.title10
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms10_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title11
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_3
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_4
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_5
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms11_6
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        imprint.title12
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms12_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        Eisbach Riders 
        <br />
        Robin Eisenhardt & Michael Schmidt GbR 
        <br />
        Lothstr. 72 
        <br />
        80797 München 
        <br />
        Deutschland 
        <br />
        E-Mail: business@eisbach-riders.com
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-rljeht"
      >
        imprint.terms12_2
      </p>
      <h5
        class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-1j3kb4i"
      >
        imprint.credits
      </h5>
      <a
        href="https://unsplash.com/@thefoxicon?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Mike Fox"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Mike Fox
        </span>
      </a>
      <a
        href="https://unsplash.com/@jeremybishop?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Jeremy Bishop"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Jeremy Bishop
        </span>
      </a>
      <a
        href="https://unsplash.com/@timmarshall?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Tim Marshall"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Tim Marshall
        </span>
      </a>
      <div>
        Icons made by
         
        <a
          href="https://www.freepik.com/"
          title="Freepik"
        >
          Freepik
        </a>
         
        from
         
        <a
          href="https://www.flaticon.com/"
          title="Flaticon"
        >
          www.flaticon.com
        </a>
         
        is licensed by
         
        <a
          href="http://creativecommons.org/licenses/by/3.0/"
          target="_blank"
          title="Creative Commons BY 3.0"
        >
          CC 3.0 BY
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
